src/__tests__/baseFields.int.test.ts

```diff
@@ -59,28 +58,13 @@ describe('/canonicalFields', () => {
         );
     });
 
-    it('should error if the database returns an unexpected data structure', async () => {
```

Comment:
As we have talked about, I took out the implementation validation since we don't want to validate the shape of database returns.

That said, we are now no longer safely protecting against an issue where our SQL has not been properly formatted to select the correct shape of data.  This means we *could* introduce a bug that violates our documented API schema and our automated tests are no longer capturing that possibility.